posts/2020-09-01-jakarta-microprofile-beta-20.0.0.10.adoc

```diff
@@ -0,0 +1,336 @@
+at link:https://github.com/jaegertracing/jaeger-client-java/blob/v1.2.0/jaeger-core/README.md[jaeger-client-java readme].
+
+For the `JAEGER_PASSWORD` environment variable, the password can be encoded using the securityUtility command.
+
+Depending on Jaeger’s sampling settings `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM`, 
+Jaeger may not report every spans created by the applications.
+
+For Zipkin, link:https://github.com/WASdev/sample.opentracing.zipkintracer[see here].
+
+Define your application in the server.xml:
+[source, xml]
+----
+<webApplication location="yourapp.war" contextRoot="/yourapp">
+    <!-- enable visibility to third party APIs -->
+    <classloader apiTypeVisibility="+third-party" />
+</webApplication>
+----
+
+Once you have hit some JAX-RS endpoints of your application, you should be able to find spans in the user interface of your tracing backend.
+
+More information:
+
+* link:https://github.com/eclipse/microprofile-opentracing[MicroProfile OpenTracting 2.0]
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+The main change visible to developers in the Jakarta EE 9 planned release is the names of packages changing to accomodate the new `jakarta.*` namespace. In this Open Liberty beta, we have more Jakarta EE 9 features with their name change completed. We've also made excellent TCK progress with these features.
+
+This Open Liberty beta introduces the following Jakarta EE 9 features which now possess their all-new Jakarta EE 9 package names:
+
+* jaspic-2.0
+* jacc-2.0
+* jpa-3.0 (The jpa-3.0 feature includes Eclipselink 3.0-RC1.)
+
+These join the Jakarta EE 9 features in link:https://openliberty.io/blog/?search=beta&key=tag[previous Open Liberty betas]:
+
+* Jakarta XML Binding 3.0 (`jaxb-3.0`)
+* Jakarta Managed Bean 2.0 (`managedBeans-2.0`)
+* Concurrency 2.0 (`concurrent-2.0`)
+* Jakarta Enterprise Beans Home 4.0 (`ejbHome-4.0`)
+* Jakarta Enterprise Beans Lite 4.0 (`ejbLite-4.0`)
+* Bean Validation 3.0 (`beanValidation-3.0`)
+* CDI 2.0 (`cdi-2.0`)
+* WebSocket 2.0 (`websocket-2.0`; currently the integration with CDI is not completed)
+* JPA 3.0 (`jpa-3.0`)
+* JDBC 4.2 & 4.3 (`jdbc-4.2` & `jdbc-4.3`)
+* JTA 2.0 (`transaction-2.0`)
+* JSON-B 2.0 (`jsonb-2.0`)
+* JSON-P 2.0 (`jsonp-2.0`)
+* Servlet 5.0 (`servlet-5.0`)
+* JSP 3.0 (`jsp-3.0`)
+* Expression Language 4.0 (`el-4.0`)
+
+==== Try it now
+
+To try out these Jakarta EE 9 features on Open Liberty in a lightweight package, just update your build tools to pull the Open Liberty Jakarta EE 9 Beta Features package instead of the main release. The beta works with Java SE 14, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty.beta</groupId>
+    <artifactId>openliberty-jakartaee9</artifactId>
+    <version>20.0.0.RELEASE_VERSION-beta</version>
```

Comment:
add the version number so it reads as `20.0.0.10-beta`

Reply:
done